local-tests/chainrunner/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jsonrpcclient==4.0.2
```

Comment:
move this file to the root directory, i.e. directly in local-tests/; (it is also needed for the scripts there)

Reply:
Done